Core/StorageCache.swift

```diff
@@ -64,17 +64,20 @@ public class StorageCache: StorageCacheUpdating {
             return fileStore.persist(data as? Data, forConfiguration: configuration)
             
         case .trackerDataSet:
-            if fileStore.persist(data as? Data, forConfiguration: configuration) {
-                if TrackerDataManager.shared.reload(etag: etag) != .downloaded {
-                    Pixel.fire(pixel: .trackerDataReloadFailed)
-                    return false
-                }
-                return true
+            if fileStore.persist(data as? Data, forConfiguration: configuration),
+               TrackerDataManager.shared.reload(etag: etag) != .downloaded {
+                Pixel.fire(pixel: .trackerDataReloadFailed)
+                return false
             }
-            return false
+            return true
             
-        case .temporaryUnprotectedSites:
-            return fileStore.persist(data as? Data, forConfiguration: configuration)
```

Comment:
Do you know if the old `temporaryUnprotectedSites` will get cleaned up? Ideally we should remove it if it isn't being used any more.